src/ResourceManagement/ApiApp/ApiAppManagement/ApiAppManagement.csproj

```diff
@@ -0,0 +1,108 @@
+  </PropertyGroup>
+  <Import Project="..\..\..\..\tools\Library.Settings.targets" />
+  <ItemGroup>
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Xml.Linq" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Xml" />
+    <Reference Include="Hyak.Common">
+      <HintPath>$(LibraryNugetPackageFolder)\Hyak.Common.1.0.2\lib\portable-net403+win+wpa81\Hyak.Common.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.Azure.Common">
+      <HintPath>$(LibraryNugetPackageFolder)\Microsoft.Azure.Common.2.1.0\lib\portable-net45+wp8+wpa81+win\Microsoft.Azure.Common.dll</HintPath>
+    </Reference>
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Generated\ApiAppManagementClient.cs" />
+    <Compile Include="Generated\ApiAppManagementClientExtensions.cs" />
+    <Compile Include="Generated\ApiAppOperations.cs" />
+    <Compile Include="Generated\ApiAppOperationsExtensions.cs" />
+    <Compile Include="Generated\IApiAppManagementClient.cs" />
+    <Compile Include="Generated\IApiAppOperations.cs" />
+    <Compile Include="Generated\ITemplateOperations.cs" />
+    <Compile Include="Generated\Models\ApiApp.cs" />
+    <Compile Include="Generated\Models\DeploymentTemplateMetadata.cs" />
+    <Compile Include="Generated\Models\GetDeploymentMetadataRequest.cs" />
+    <Compile Include="Generated\Models\GetDeploymentTemplateMetadataResponse.cs" />
+    <Compile Include="Generated\Models\LengthConstraint.cs" />
+    <Compile Include="Generated\Models\ListApiAppsResponse.cs" />
+    <Compile Include="Generated\Models\MicroserviceApi.cs" />
+    <Compile Include="Generated\Models\MicroserviceAuthentication.cs" />
+    <Compile Include="Generated\Models\MicroserviceLink.cs" />
+    <Compile Include="Generated\Models\MicroserviceMetadata.cs" />
+    <Compile Include="Generated\Models\MicroservicePackage.cs" />
+    <Compile Include="Generated\Models\ParameterConstraints.cs" />
+    <Compile Include="Generated\Models\ParameterMetadata.cs" />
+    <Compile Include="Generated\Models\RangeConstraint.cs" />
+    <Compile Include="Generated\Models\ResourceReference.cs" />
+    <Compile Include="Generated\Models\StatusCheckEntry.cs" />
+    <Compile Include="Generated\Models\StatusLevel.cs" />
+    <Compile Include="Generated\Models\TemplateParameterType.cs" />
+    <Compile Include="Generated\Models\UIHint.cs" />
+    <Compile Include="Generated\TemplateOperations.cs" />
+    <Compile Include="Generated\TemplateOperationsExtensions.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+  </ItemGroup>
+  <ItemGroup />
+  <Import Project="$(MSBuildExtensionsPath32)\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.CSharp.targets" Condition=" '$(LibraryFxTarget)' == 'portable' " />
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
```

Comment:
sorry, github was removing the improperly formatted code stings :) fixed in the comments